src/js/post-911-gib-status/reducers/index.js

```diff
@@ -5,12 +5,15 @@ import {
   BACKEND_SERVICE_ERROR,
   GET_ENROLLMENT_DATA_FAILURE,
   GET_ENROLLMENT_DATA_SUCCESS,
-  NO_CHAPTER33_RECORD_AVAILABLE
+  NO_CHAPTER33_RECORD_AVAILABLE,
+  SERVICE_UP_STATES,
+  SET_SERVICE_UP
 } from '../utils/constants';
 
 const initialState = {
   enrollmentData: null,
-  availability: 'awaitingResponse'
+  availability: 'awaitingResponse',
+  serviceUp: SERVICE_UP_STATES.unrequested
```

Comment:
We could default this to `pending` since we call it immediately anyhow. I don't have strong feelings about this.